tests/roles/ceph_migrate/tasks/firewall.yaml

```diff
@@ -10,29 +10,76 @@
-      -A INPUT -p tcp -m tcp --dport 8090 -m conntrack --ctstate NEW -m comment --comment "101 ceph_rgw ipv4" -j ACCEPT
-      -A INPUT -p tcp -m tcp --dport 8989 -m conntrack --ctstate NEW -m comment --comment "102 ceph_rgw ipv4" -j ACCEPT
-      -A INPUT -p tcp -m tcp --dport 3300 -m conntrack --ctstate NEW -m comment --comment "110 ceph_mon ipv4" -j ACCEPT
-      -A INPUT -p tcp -m tcp --dport 6789 -m conntrack --ctstate NEW -m comment --comment "111 ceph_mon ipv4" -j ACCEPT
-      -A INPUT -p tcp -m tcp --dport 2049 -m conntrack --ctstate NEW -m comment --comment "111 ceph_nfs ipv4" -j ACCEPT
-      -A INPUT -p tcp -m tcp --dport 12049 -m conntrack --ctstate NEW -m comment --comment "111 ceph_nfs_backend ipv4" -j ACCEPT
-      -A INPUT -p tcp -m tcp --dport 6800:7300 -m conntrack --ctstate NEW -m comment --comment "112 ceph_mds_mgr ipv4" -j ACCEPT
-  loop: "{{ firewall_path }}"
+- name: Manage Ceph iptables rules
+  when: ceph_firewall_type == "iptables"
+  block:
+    - name: Ceph Migration - Apply the Ceph cluster rules (iptables)
+      delegate_to: "{{ node }}"
+      become: true
+      ansible.builtin.blockinfile:
+        marker_begin: "BEGIN ceph firewall rules"
+        marker_end: "END ceph firewall rules"
+        path: "{{ item }}"
+        block: |
+          -A INPUT -p tcp -m tcp --dport 8080 -m conntrack --ctstate NEW -m comment --comment "100 ceph_rgw ipv4" -j ACCEPT
+          -A INPUT -p tcp -m tcp --dport 8090 -m conntrack --ctstate NEW -m comment --comment "101 ceph_rgw ipv4" -j ACCEPT
+          -A INPUT -p tcp -m tcp --dport 8989 -m conntrack --ctstate NEW -m comment --comment "102 ceph_rgw ipv4" -j ACCEPT
+          -A INPUT -p tcp -m tcp --dport 3300 -m conntrack --ctstate NEW -m comment --comment "110 ceph_mon ipv4" -j ACCEPT
+          -A INPUT -p tcp -m tcp --dport 6789 -m conntrack --ctstate NEW -m comment --comment "111 ceph_mon ipv4" -j ACCEPT
+          -A INPUT -p tcp -m tcp --dport 2049 -m conntrack --ctstate NEW -m comment --comment "111 ceph_nfs ipv4" -j ACCEPT
+          -A INPUT -p tcp -m tcp --dport 12049 -m conntrack --ctstate NEW -m comment --comment "111 ceph_nfs_backend ipv4" -j ACCEPT
+          -A INPUT -p tcp -m tcp --dport 6800:7300 -m conntrack --ctstate NEW -m comment --comment "112 ceph_mds_mgr ipv4" -j ACCEPT
+      loop: "{{ ceph_iptables_path }}"
 
-- name: Ensure firewall is enabled/started
-  when: ceph_firewall | bool | default(false)
-  delegate_to: "{{ node }}"
-  become: true
-  ansible.builtin.systemd:
-    name: iptables
-    state: restarted
-    enabled: true
+    - name: Ensure firewall is enabled/started - iptables
+      when: ceph_firewall_enabled | bool | default(false)
+      delegate_to: "{{ node }}"
+      become: true
+      ansible.builtin.systemd:
+        name: iptables
+        state: restarted
+        enabled: true
+
+
+- name: Manage Ceph nftables rules
+  when: ceph_firewall_type == "iptables"
```

Comment:
is it meant to say `when: ceph_firewall_type == "nftables"` here?

Reply:
oh yes, good catch. I'll run testproject again with the updated version (even though by mistake I successfully executed both parts)